test_unit.py

```diff
@@ -902,7 +904,7 @@ def test_without_system_profile(
         }
         host_schema.return_value.load.return_value.data = input
```

Comment:
Here, the variable name really is _input_. It works, but it is a bad practice. (And it’s probably my fault it’s like that. 😞) I’d like to ask you for a favor of renaming those to _host_input_ throughout the whole file. Thanks!

```suggestion
        host_schema = Mock(**{"return_value.load.return_value.data": host_input})
```